### ISSUE TITLE
Suggestion: Generic life icons

### ISSUE THREAD

**Issue by Edinho367** [Rhodney/keepass-icons#4]
Hello Rhodney!

I just downloaded your icon packs, and they are incredible! 

Thank you very much for the trouble of making them available to us, because our databases get way more visually organized using the correct icons! It demands some work to assign them to each entry but it pays 10 times making it very easy to find what we need!

I am a keepass enthusiast, and I store most of my life info there (personal documents data, logins, medical history, spreadsheet passwords, social security numbers of mine and my family, etc) so today, when i stumbled upon your icon sets, it made me very happy.

I think your icons will be more than enough for 95% of people who search "icons keepass" on google; although I'd like to suggest you to expand, including **Generic life icons**. 

What would that be? Well, here some examples:
* A market cart for physical stores-related info
* University /graduation hat for course-related info
* Popcorn / clapper for movies sites (each cinema has its own nowadays)
* Some sports icons (swimming guy / soccer ball or football) for related subjects, like soccer club id, or club
* Bus / train for logins used to buy tickets
* Medicine pills or something similar to pharmacy data
* A car / motorbike for parking data
* House / industrial building for electronical locks passwords

Well, i think you already got the idea by the examples. Maybe, to make it easier and faster, you could even convert some emojis into .ico files to represent some of this situations.

In summary, the suggestion would be to add a folder with generic symbols of things we use in 'real life' encouraging the use of keepass not just for 'online things' but general storage of sensitive info and passwords of any kind.

**Comment by Rhodney** [comment]
Hi Edinho!

I'm really happy to hear that you liked this icon collection so much and that it was so useful for you.

Although I'm short on time, I'll keep this issue opened for now and I'll fulfill your requests when I can, even if it takes a while. In the meantime, keep an eye on the notifications you receive about this issue and the repository in general, because I'll be addressing it in several separate commits.

**Comment by Flintii** [comment]
Hi Guys and hi Rhodney!
I find this Plugin is realy cool and I like it. 
By the way are you also a KeePass user and do you actually use Keepass privately?
I would be happy if you answer me.

**Comment by Rhodney** [comment]
Hi, @Greatcode-tech !
Yes, I'm a keepass user and I use it privately.
I like the idea of using an image pack like this instead of plugins that automatically download images from the internet.

The advantages of this image pack over plugins are:
- You need internet connection only once to download this image pack.
- The size of each icon file is extremely optimized to take up less space in your database.
- It's not exposed to potential plugin security vulnerabilities/malicious code.

Disadvantages:
- It doesn't have all the images.
- You need to manually select the image from the pack when creating a new entry in keepass.
